src/components/ContributionAssistantPriceFormCard.vue

```diff
@@ -0,0 +1,84 @@
+<!-- eslint-disable vue/no-mutating-props -->
+<template>
+  <v-card
+    class="mb-4"
+    title="Label"
+    prepend-icon="mdi-tag-outline"
+    height="100%"
+    style="border: 1px solid transparent"
+    :color="productPriceForm.processed ? 'success' : ''"
+  >
+    <v-divider />
+    <v-img
+      height="200px"
+      :src="productPriceForm.proofImage"
+      contain
+    />
+    <v-card-text>
+      <ProductInputRow :productForm="productPriceForm" :disableInitWhenSwitchingModes="true" @filled="productFormFilled = $event" />
```

Comment:
maybe the mode init could be done beforehand ?
asking if its a photo of barcodes, or categories ?